### ISSUE TITLE
local installation of EuclidTheGame

### ISSUE THREAD

**Issue by rinaldi6109** [kasperpeulen/euclidthegame#363]
I wish to show EuclidTheGame to my students in a classroom where the wi-fi connection is very very poor. The loading of geogebra app is very slow and several times it fails. 
Would it be possible to "install" the game in a local environment? I could install it on a pc and then share it via wi-fi directly from the pc.

**Comment by quadboston** [comment]
Yes. While the source code found on this site is intended primarily for the purpose of development, it can also be used to run the game locally on your computer. But it wouldn't be a regular consumer install as you're used to. For one thing, I think you'll need to set up a local web server. (I don't know how hard that is on Windows; on a Mac it's easy enough). Let me know if you're still interested, and if you're talking Mac or Windows.
